src/modules/create_user/app/create_user_viewmodel.py

```diff
@@ -0,0 +1,28 @@
+from src.shared.domain.entities.user import User
+from src.shared.domain.enums.restaurant_enum import RESTAURANT
+from src.shared.domain.enums.role_enum import ROLE
+
+
+class CreateUserViewmodel:
+    user_id: str
+    name: str
+    email: str
+    restaurant: RESTAURANT = None
+    role: ROLE 
+
+    def __init__(self, user: User):
+        self.user_id = user.user_id
+        self.name = user.name
+        self.email = user.email
+        self.restaurant = user.restaurant
+        self.role = user.role
+
+    def to_dict(self):
+        return {
+            'user_id': self.user_id,
```

Comment:
ao inves de retornar todos juntos, retorna assim

return {
            "user": self.user_viewmodel.to_dict(),
            "message": "the user was retrieved"
        }

sendo que o user_viewmodel.to_dict() é 

return {
            "name": self.user_name,
            "email": self.user_email,
            "role": self.role,
            "user_id": self.user_id,
            "restaurant": self.restaurant
        }

faz sentido?

Reply:
OK